VotingApplication/VotingApplication.Web/Views/Home/Index.cshtml

```diff
@@ -7,8 +7,8 @@
 <body>
     <div class="center-panel">
         <div class="body-content">
-            <h2>Where do we go for lunch?</h2>
-            <p id="expiry-date">Poll closes in: 5 hours</p>
+            <h2>{{pollName}}</h2>
```

Comment:
The question mark is going to be added at the poll creation stage (mostly for backwards compatibility here when calling up old polls).